state/machine.go

```diff
@@ -154,6 +154,10 @@ func newMachine(st *State, doc *machineDoc) *Machine {
 	return machine
 }
 
+func (m *Machine) Doc() machineDoc {
```

Comment:
Doesn't look like you've used this. We probably shouldn't export such a thing anyway.

Reply:
Sorry I'll remove this. Was for testing a constraint setup